test/parallel/test-zlib.js

```diff
@@ -202,7 +202,7 @@ Object.keys(tests).forEach(function(file) {
 });
 
 process.on('exit', function(code) {
-  console.log('1..' + done);
+  console.log('1..' + done + ' of ' + total);
```

Comment:
This is not necessary. Also, this format is Not TAP plugin parsable I think. Perhaps we can remove that line. cc @nodejs/build